src/i_oalmusic.c

```diff
@@ -46,6 +46,7 @@ static stream_module_t *all_modules[] =
 #if defined(HAVE_LIBXMP)
     &stream_xmp_module,
 #endif
+    &stream_mp3_module
```

Comment:
Do we need to have this line in `stream_modules[]` as well?

Reply:
Done in https://github.com/fabiangreffrath/woof/commit/42f8b0c0416e9ba66a907fe5c4432174a32f4560